Platform Support.md

```diff
@@ -0,0 +1,17 @@
+# Platform Support
+
+Being built on cargo and rustc, maturin is limited by [rust's platform support](https://doc.rust-lang.org/nightly/rustc/platform-support.html).
+
+**Automated tests**: On github actions, windows, mac and linux are tested, all on 64-bit x86. FreeBSD is also tested though circle ci, but might get removed at some point. Since CI is very time intensive to maintain, I'd like to stick to github action and these three platforms.
+
+**Releases**: The following targets are built into wheels and downloadable binaries:
+
+ * windows: 32-bit and 64-bit x86 as well armv7 and aarch64
```

Comment:
I don't think we have armv7 and aarch64 binaries for windows? They are for Linux.

Reply:
Yeah that was a typeo

Reply:
This still says `armv7 and aarch64` for windows.